### DISCUSSION TITLE
Definition der Einsatzbereiche

### DISCUSSION THREAD

**Discussion by fforootd** [e-id-admin/governance-sounding-board#10] [General]
Ich denke es wäre für den Erfolg einer E-ID wichtig, das zuerst deren Einsatzspektrum geklärt wird. Aktuell sehe ich keine klare Aussage was das Ziel der E-ID ist.

Folgende Einsatzbereiche sollten unterschieden werden:

- Nachweis der Identität (Mehr oder weniger die Attribute auf der ID, Pass, Ausländerausweis)
- Nutzung als Authentifizierungsmittel (PIV, X509, FIDO2, ...)
- Nachweis von (Fachlichen)Attributen (GLN, Führerausweis, ...)
- Signatur von Daten
- Verschlüsselung von Daten

In meinen Augen sollte das klare Ziel der E-ID, insbesonders in einer ersten Phase, sein das ein Nachweis einer Identitat im Digitalen Geschäft erbracht werden kann. Die Nutzung im analogen Geschäft mag des weiteren ein Nachrangiges Ziel sein.

Was eine E-ID nicht sein soll, ist ein Sammelsurium von Attributen und Nachweisen (VCs) für alle möglichen Dinge. Das heist jedoch nicht das VCs nicht an eine E-ID geknüpft werden dürfen. Des weiteren sehe ich nicht dass die E-ID als Ersatz für diverse Logins zum Einsatz kommt. Die Ausnahme mag hier das eGov Geschäft darstellen wo eine E-ID für den Login allenfalls eingesetzt werden könnte.

**Comment by MarcelWaldvogel** [comment]
Beim Nachweis der Identität sollte auch unterschieden werden, ob diese gegenüber Behörden oder Privaten vorgebracht werden. Insbesondere bei Letzteren sollten sowohl die Verifizierung als auch die Attribute möglichst stark eingeschränkt werden. Analog zum nPA in D sollte auch
1. gesetzlich klar geregelt sein, wer was damit darf und was nicht (und das wenn möglich auch technisch durchgesetzt werden, damit auch sensible Einsatzbereiche möglich sind) und 
2. ein Alternsnachweis ohne Preisgabe von weiteren Details erfolgen können.

**Reply by fforootd** [reply]
Danke für die Ergänzung, damit bin ich absolut einverstanden. Es soll die Selektive weitergaben von Attributen möglich sein. Zero-Knowledge für das Alter wäre sicher auch noch eine gute Sache.

**Reply by grt-dti** [reply]
Das Problem ist hier ein ganz einfaches: wenn man es den Inhabern nicht gesetzlich erlaubt (oder umgekehrt den "Anderen" verbietet), den "Interessenträgern" die E-ID vorzuenthalten, werden diese einfach die Leistung verweigern, bis die E-ID (in dem für sie gewünschten Umfang) preisgegeben ist. Das Problem ist kein technologisches, es ist ein juristisches und ökonomisches....

**Reply by christianheimann** [reply]
Das Machtgefälle existiert immer, auch in der analogen Welt: Man gibt Informationen preis, weil man danach gefragt wird - nicht weil das Gegenüber diese benötigen würde. Im digitalen Raum müsste da ja das DSG respektive bald nDSG greifen.  
_Was wäre damit eurer Ansicht nicht abgedeckt? Warum würde das nDSG ungenügend vor Missbrauch schützen?_

**Reply by grt-dti** [reply]
das DSG (egal welches) funktioniert in einem Machtgefälle nicht, weil es von einem Verhältnis gleichberechtigter Partner ausgeht - und dieses haben wir ja in dem Szenario genau nicht (btw. das Szenario ist ein (quasi)monopolistisches Angebot ohne nennenswerte Alternativen).
Der Unterscheid zur realen Welt ist ein einfaches: 
In der virtuellen Welt ist die Präsentation des Plastik-Ausweises ein aufwändiger Akt und dem Wert der Daten selten angemessen.In der realen Welt ist jedem klar, dass er die Informationen auf dem Ausweis nicht "schnell genug" kopieren kann, damit diese einen Wert bekommen.
Mit der Elektronifizierung des Ausweises senken wir die Hürde für die massive Datensammlung aber extrem (und zahlen mit unseren Steuergeldern auch noch dafür) und dann "lohnt" sich die "Erzwingung der Herausgabe" plötzlich.
Hier darf man eines nicht ausser Acht lassen:
Umso monopolistischer das Angebot desto niedriger die zusätzlichen Opportunitätskosten (also dass ein Nutzer wegen genau dieser "Erzwingung" eben doch nicht "bei mir" sondern der Konkurrenz konsumiert).

@christianheimann das heutige DSG kann uns davor nicht schützen; nicht einmal die DSGVO vermag das wirklich und die ist dem nDSG einen riesen Schritt voraus.
ABER die deutsche Lösung des nPA (jeder Konsument des nPA muss sich registrieren, seine Begründung für den Bezug offen legen und um "Autorisation bitten") kann's auch nicht wirklich sein - das ist zwar sicher die "beste" Variante, um dem Machtgefälle etwas wirksames entgegen zu setzen, aber das bremst die Nutzung natürlich extrem ein. Darum hat die DSGVO ja genau den Mittelweg (Offenlegung der Datenverarbeitung und Recht auf Korrektur und Löschung) versucht - dann bleibt die Autorisation immer noch beim Dateninhaber/in, aber die/der kann zumindest eine nennenswert fachlich fundierte Autorisationsentscheidung treffen und seine Meinung nachträglich ändern.
Wenn man dem Machtgefälle aber wirklich etwas entgegen setzen will, braucht der Einzelne einen "Meinungsverstärker" und davon haben wir (dummerweise) mit der Erfindung der sozialen Netzwerke mittlerweile zu viele. Man könnte auch sagen "_Die Regulation ist uns aus den Händen geglitten._"

**Reply by datenreisen** [reply]
Was im **E-ID-Gesetz** wichtig ist, ist eine strenge Zweckbindung der Daten vorzusehen, die durch «das Ausweisen» bei der anfragenden Instanz (beim «Verifyer») anfallen (Inhaltsdaten, Metadaten, Profile). Diese dürfen nur für die notwendigen und offensichtlichen Zwecke - und nicht über das Kleingedruckten für alles Mögliche - angefragt und verwendet werden. Damit soll verhindert werden, dass wir uns für alltägliche Dinge plötzlich «ausweisen» müssen und durch die E-ID weiter gläsern werden.

**Reply by grt-dti** [reply]
> Was im **E-ID-Gesetz** wichtig ist, ist eine strenge Zweckbindung der Daten vorzusehen, die durch «das Ausweisen» bei der anfragenden Instanz (beim «Verifyer») anfallen (Inhaltsdaten, Metadaten, Profile). Diese dürfen nur für die notwendigen und offensichtlichen Zwecke - und nicht über das Kleingedruckten für alles Mögliche - angefragt und verwendet werden. Damit soll verhindert werden, dass wir uns für alltägliche Dinge plötzlich «ausweisen» müssen und durch die E-ID weiter gläsern werden.

ich kann leider nur einen Smiley vergeben - wenn ich könnte, wären es 100......
Herausforderung wird die Umsetzung dieser Erwartung sein.

**Reply by KampfCaspar** [reply]
Neben der Weiterverwendung ist auch die Pflicht zum Ausweisen (bzw. Verhinderung derselben) zu legiferieren. Nur wo Identität objektiv zwingend ist, darf sie verpflichtend ieS. sein (zB. eGov).

Was zu diskutieren wäre: Dürfen mit dem Ausweisen Vorteile verbunden sein (z.B. Lieferung auf Rechnung nur mit Identifikation)?

**Reply by fforootd** [reply]
Die incentivierung über "nudges" finde ich schlecht. Da so ein Personkreis diskriminiert wird welcher zum Beispiel die ID nicht oder nur schwer nutzen kann (Behinderungen, Zugang zur Technologie,...)

Ansonsten stimme ich zu 😉

**Reply by grt-dti** [reply]
> Was zu diskutieren wäre: Dürfen mit dem Ausweisen Vorteile verbunden sein (z.B. Lieferung auf Rechnung nur mit Identifikation)?

Naja; Frage wäre, darf man "Nicht-Ausgewiesenen" diesen Vorteil verwehren? Und hier kommt dann neben der juristischen Keule vor allem die moralische ins Spiel. Im Beispiel "Rechnungskauf" kann man problemlos moralisch-integer argumentieren, dass eine Identifikation das Schadensrisiko signifikant reduziert und damit den im Preis eingerechneten Risikoaufschlag für "alle" reduziert - Voraussetzung wäre aber, dass dieser "Gemeinschaftsnutzen" (aufgrund der erzwungenen Identifikation) an "alle" verteilt wird und nicht als Marge (alleinig) beim Händler hängen bleibt. Einen Verstoss dagegen kann man aber einerseits gar nicht identifizieren und andererseits nur mit Verzicht oder Anbieterwechsel ahnden.
Dazu kommt ein grundlegendes Dilemma: Eine Wirkung hat immer (mindestens) eine Ursache, aber eine Ursache kann mehrere Wirkungen haben. Im Beispiel: den Risikoaufschlag für "alle" kann man auch einfach durch moralische Ehrlichkeit reduzieren; dafür braucht's nicht zwingend Identifikation.....

**Comment by KampfCaspar** [comment]
Vielleicht sollten wir diskutieren, welche Einsatzbereiche wir sehen? Ich habe mir zur Abstimmung und auch im neuen Projekt schon den Mund fusselig geredet, dass eine eID per se kein Login ist. Ob man darauf aufbauend parallel ein 'Schweizer Loginsystem' mit IdP aufbauen will, kann man diskutieren - auch wenn ich dagegen wäre. Auch eine FIDO2-Funktionalität hat grundsätzlich mal nichts damit zu tun; aber da sehe ich einen grossen Vorteil, wenn man das breite Publikum mit einer Token-Funktionalität ausstattet. Die muss trotzdem getrennt von der eID laufen, also ohne eID-Zusatz nutzbar sein.
 
Für ein eID-Ökosystem sehe ich tatsächlich ein 'Sammelsurium beliebiger Attribute', mit verschiedenen Trust Anchors und genau definierbarer Freigabe einzelner Attribute. Das besondere an den 'Pass-Attributen' ist nur der staatliche Aussteller.  Attribute sollten auch über verschiedene Issuer abhängig sei können - denn genau darin liegt ja der Vorteil eines Ökosystems:

Und: wenn man nicht parallel on- und offline-Use vorsieht, vergibt man sich IMHO nützliche Anwendungsszenarien. Natürlich können e-Attribute nur digital überprüft werden, es braucht also auch offline eine 'Check-App'.

- Dem Spirituosen-Webshop will ich nachweisen, dass der von mir dort angegebene Empfänger >18 Jahre alt ist. Ich muss dazu keinen Account eröffnen und kein staatliches Login nutzen - ja es braucht noch nicht einmal eine Übermittlung des Namens oder des Geburtsdatums von der ID zum Shop. Der Shop kann anfragen, ob der eingegebene Name stimmt und ich >18 bin.
- Am Club-Eingang reicht 'Bildnis' und >18 - es braucht kein Geburtsdatum, Adresse, Namen, ...
- Am Postschalter will ich belegen, dass mein Bildnis zu dem Namen gehört, der auf dem Abholzettel steht (es braucht kein Geburtsdatum, ...). Wenn die Abholeinladung dann einmal elektronisch zugestellt wird, braucht es vielleicht gar nichts mehr - selbst wenn ich einem Kollegen eine Einzelermächtigung zur Abholung ausstelle.
- Der eFührerausweis (staatlich) aber auch das eTicket (halb-privat) oder der eMitarbeiterausweis (privat) kann auf Attributen der Kern-eID (Name, Bild) aufbauen. Damit würde eine reine Online-Ausstellung ermöglicht - gleichzeitig kann ich bei der Strassen-/Billet-/Einlasskontrolle offline im ersten Schritt nur mein 'Bildnis' und meine Berechtigung preisgeben.

**Reply by fforootd** [reply]
Ja die Diskussion der Use-case ist sicher wichtig. Ohne das wird nie klar werden wozu die EID geplant ist.

> Vielleicht sollten wir diskutieren, welche Einsatzbereiche wir sehen? Ich habe mir zur Abstimmung und auch im neuen Projekt schon den Mund fusselig geredet, dass eine eID per se kein Login ist. Ob man darauf aufbauend parallel ein 'Schweizer Loginsystem' mit IdP aufbauen will, kann man diskutieren - auch wenn ich dagegen wäre. Auch eine FIDO2-Funktionalität hat grundsätzlich mal nichts damit zu tun; aber da sehe ich einen grossen Vorteil, wenn man das breite Publikum mit einer Token-Funktionalität ausstattet. Die muss trotzdem getrennt von der eID laufen, also ohne eID-Zusatz nutzbar sein.

Das kenne ich nur zu gut 😁 

> Für ein eID-Ökosystem sehe ich tatsächlich ein 'Sammelsurium beliebiger Attribute', mit verschiedenen Trust Anchors und genau definierbarer Freigabe einzelner Attribute. Das besondere an den 'Pass-Attributen' ist nur der staatliche Aussteller. Attribute sollten auch über verschiedene Issuer abhängig sei können - denn genau darin liegt ja der Vorteil eines Ökosystems:

Die selektive Freigabe ist definitv ein wichtiges Element. 
Ich denke aber die EID soll in andern Trustsystemen als Identifier genutzt werden können. Dazu können eigene Ökösysteme geschaffen werden welche auf die EID verlinken. Das würde verhindern dass die EID zur wilden Sammlung von Attributen wird und gleichzeitig könnte jedes Ökosystem selber die Governance übernehmen. Wichtig dabei wäre jedoch das Tracking zwischen Ökosystemen zu erschweren. Dies liese sich jedoch gut mit einer "pairwise" ID o.ä. lösen.

IMO ist also EID == digitale Form des Passes / ID / Ausländerausweis darauf können zum beispiel die ÖV Betreiber Abos und Rabattkarten aus deren eigenen Systemen verlinken. Das ermöglich auch das Anwendungsfälle welche nicht SSI sind einfach integrierbar sind.

**Reply by ii-nik** [reply]
Genau, das ist auch meine Meinung. Damit ist die eID eigentlich "nur" ein VC (Verifiable Claim), welches aber (wie oben auch schon erwähnt) die selektive Weitergabe von Attributen erlauben muss, sowie auch Zero Knowledge fähig sein muss.

Ich würde aber noch einen Schritt weiter gehen. Auch das Linking von der eID zu anderen Nachweise wie Führerausweis, ÖV-Abo finde ich heikel und ich würde aus folgenden Gründen:

* Was ist, wenn die eID ungültig wird (warum auch immer): was passiert dann mit den verlinkten VCs?
* Was ist, wenn jemand nicht die eID, sondern einen anderen Identitätsnachweis vorzeigt (Ausländischer Pass, Plastik-ID), dann den Führerausweis aber doch als VC haben möchte?

Daher: Meine Meinung ist, dass die Herausgeber (Issuer) eines VC (ÖV-Abo, Führerausweis) zur Identitätsprüfung schon die eID verwenden dürfen. Wenn der Issuer dann aber ein VC ausstellt, übernimmt der Issuer für das ausgestellte VC und den korrekten Ablauf der Identitätsprüfung die Verantwortung. Im VC sollten  sich keine Spuren der ursprünglichen Identifikation mehr finden (auch eine Art Datensparsamkeit ;-))

Dementsprechend auch: Ein Verifier (z.B. die Polizei, welche einen Führerausweis prüft), würde nun vom Owner / Holder einfach 2 Nachweise verlangen: Einerseits den Führerausweis und andererseits (falls nötig) einen Identitätsnachweis.

**Reply by KampfCaspar** [reply]
Wie gesagt, ich bin nicht abgeneigt, zusammen mit der eID einen Identifier (z.B. FIDO-Token) anzubieten. Die Verbindung zur eID würde ich aber nicht machen. Natürlich kann ich überall, wo ich den Identifier verwende, freiwillig auch VCs freigeben. Das sind aber getrennte Funktionen.

**Reply by fforootd** [reply]
IMO ein guter Prozess wäre wenn der User mit seinem Pass/ID/Ausländerausweis sich selber sein Wallet enrollen könnte wo er eine Kopie seiner Attribute i, sicheren Element des Devices speicher kann (inkl. einem frischen Signatur key). Mit diesem Wallet könnte der User dann Attribute an Parteien bereitstellen und zur Authentifzierung von niedrigen LoA stufen würde FaceID/Fingerprint/Pin reichen und bei höheren die ID mit NFC (evtl. FIDO2) zum Einsatz kommen.

**Reply by grt-dti** [reply]
@KampfCaspar Es ist müssig, dem Ausweis die Login-Funktion abzusprechen - rein formal mag das stimmen, aber für die Frage der Wirkung ist es egal.
Damit man den Einsatz des Ausweises (also das "Ausweisen") rechtlich verbindlich gestalten kann, MUSS es einen Login-Vorgang in die zur E-ID gehörige Identität geben - man MUSS die Inhaberschaft der E-ID beweisen können (und zwar auf dem Verlässlichkeitsniveau auf welchem der UseCase die Verbindlichkeit definiert hat). --- ich hoffe, da sind wie uns alle einig drüber

Spätestens seit der "Erfindung der IAM Föderation" ist es möglich den Nachweis der Inhaberschaft über eine Identität zu delegieren. IdP-Fetischisten nennen dies einfach "Login in einen SAML- oder OIDC-IdP"; SSI-Fetischisten nennen es "Ausweisen mittels verifiable credential" - am Ende ist das alles aber nichts anderes als eine delegierte Verifikation eines Identitätsnachweises (den der Anwender / Inhaber) selbst in der Hand hat.

Man sollte also anstelle "Ausweis aber kein Login" lieber sagen "ein auf Ausweis upgraded Login" - dann wird aber auch klar, das es den ganzen FIDO-Kram für "E-ID-onboarded Identitäten" überhaupt nicht mehr braucht, weil die Identitätsnachweisfunktion der E-ID auch problemlos weiterhin verwendet werden kann.....

**Reply by christianheimann** [reply]
Login sehen wir vom Projektteam als _eine Anwendungsmöglichkeit_ der E-ID. Genau so gut sollte eine E-ID aber auch in einem "Login-freien" Prozess verwendet werden können; ein bisschen, wie wenn man bei einem Webshop als "Gast bestellt" und am Ende mit einer Kreditkarte bezahlt, könnte es in Prozessen einen Schritt "Identifizieren" geben, bei dem man die E-ID resp. die nötigen Attribute präsentiert. @KampfCaspar hat das schon schön formuliert oben.

**Reply by grt-dti** [reply]
das ist ja eben der Trugschluss - es gibt keinen "Login-freien" Prozess. Damit ich die E-ID einsetzen kann, MUSS ich die Inhaberschaft beweisen und dafür muss ich mich mindestens in meinem E-ID-Speicher (nennen wir es einfacherweise "Wallet App") einloggen.
Und ehe wir nun frohlocken ob eines "Login-freien" Gastbestellprozesses - auch das ist ein Login in die Bestellanwendung. Auch dort bin ich spätestens seit der Bekanntgabe meiner E-ID kein Gast (anonymous) mehr und ich besitze eine "Arbeitssitzung" (nennen wir es im Shop mal "Warenkorb") - das ist (nach meinem Verständnis zumindest) genau das Wesen eines Login's

**Comment by ii-nik** [comment]
In diesem Zusammenhang finde ich noch etwas anderes wichtig: Wir müssen den Begriff eID noch schärfen.

1. Ist die eID nur ein VC?
2. Oder ist mit eID auch die Infrastruktur (Netzwerk, Bundes-Wallet) mitgemeint?
3. Oder ist  damit das ganze Öko-System mitgemeint
4. Oder was anderes?

Meine Meinung ist, dass 1, die richtige Definition ist. 

Bei der Infrastruktur / Bundes-Wallet / Ökosystem würde ich von "vertrauenswürdige Plattform für digitale Nachweise" sprechen (Da gibt es sicher noch knackigere Bezeichnungen, aber auf die Schnelle fällt mir grad nichts ein). Damit werden auch die Ambitionslevel irgendwie hinfällig. Es sind einfach unterschiedliche Use Cases, die gleichberechtigt auf derselben Plattform laufen.  (BTW: Die Ambitionslevel stören mich von Anfang an, ein höherer Level heisst nicht, dass es besser / schwieriger zu erreichen ist, aber das ist eine andere Geschichte).

Was denkt ihr? / Was denken Sie?

**Reply by fforootd** [reply]
IMO ist die eID ein Digitaler Nachweise einer Identität (Zertifikat und/oder VC) bzw. deren Personenbezogener Attribute

**Reply by christianheimann** [reply]
Ich bestätige, dass 1. auch unsere Sicht  des Projektteams widerspiegelt: die E-ID ist das vom Bund ausgestellte Verifiable Credential, welches erlaubt, die eigene Identität nachweisen zu können.  
Ebenso teile ich die Sicht auf die erwähnte "vertrauenswürdige Plattform für digitale Nachweise".

- Wallet ist ein Container, um Verifiable Credentials zu empfangen, abzulegen und zur Präsentation zu bringen.
- Registry ist das Register, um die Vertrauensanker von Herausgeberinnen (Issuern) zu setzen (z.B. eine Art dynamisches Public Key Directory)
- Ökosystem ist der Wunschgedanke, viele Teilnehmer im System zu haben: Issuer, Verifier, Holder mit unterschiedlichsten digitalen Nachweisen. Die Rollen darin sind fliessend, da eine Organisation zuerst als Issuer auftreten kann, in einem anderen Prozess aber VCs als Verifier nutzen kann. Dazu kommt der Aspekt, dass sich die digitale Mündigkeit resp. das Wissen um das Funktionieren eines sochen Ökosystems breit etablieren kann und damit Verständnis und Vertrauen entsteht.

**Reply by fforootd** [reply]
Ich bleibe bei meiner Meinung dass der Fokus der E-ID alleinig auf der sicheren übertragung von Personenbezogenen Attributen an berechtigte Parteien gesetzt sein sollte. Dazu braucht es aber weder VC noch SSI.

VC per se ist noch nicht ansatzweise Matur und wird es vor 2025 auch nicht sein.

**Reply by grt-dti** [reply]
ohh wie sprichst du mir aus der Seele..... Das löst aber das Problem nur für die E-ID; was machen wir mit den ganzen anderen Bestätigungen in einem Ökosystem? Wenn wir diesen Digitalisierungshype weiterverfolgen, werden wir in nicht allzu ferner Zukunft auch "Eigentumsbestätigungen" für die ganzen andere "Dinge" als Identitäten brauchen. Da stimmst du mir doch zu?
Ich hab in einer schwachen Stunde tatsächlich mal darüber nachgedacht, wieso man all die ganzen nicht-staatlichen und halbstaatlichen Eigentumsbeweise nicht einfach in NFT's verpackt und über die ganz regulären (also eigentlich nicht-regulierten) Prozesse der eh schon aufgebauten blockchainbasierten virtuellen Eigentumsregister abwickelt; lass doch die BFH das Bachelordiplom in ein NFT schmeissen und das auf irgendeine Blockchain legen. Wieso soll der Staat / die staatliche Verwaltung für so einen privatwirtschaftlichen Eigentumskams Geld ausgeben und dann den Kopf hinhalten? Warum sollen wir staatliche Eigentumsbestätigungen mit privatwirtschaftlichen vermischen und uns all die Misch-Probleme einfangen?
Wenn wir schon mit SSI und Co. arbeiten wollen, warum dann nicht richtig?

**Reply by christianheimann** [reply]
@grt-dti SSI, wie es in der praktischen Umsetzung gemacht wird, schreibt zu den ausgestellten Nachweisen _keine_ Informationen in ein zentrales Register (aka Verifiable Data Registry, exkl. Revocation-Thema). Genau dadurch entsteht ein Teil des Privacy-Aspekts. Ein _digitaler Nachweis_, wie wir ihn denken, hat nicht die gleiche Charakteristik wie NFTs oder Geld, da es IMHO kein "double Spending" Problem gibt.

**Reply by christianheimann** [reply]
Es geht nicht um Vermischung @grt-dti, es geht um die Chance, stets gleiche Abläufe für den Benutzer gleich zu ermöglichen:
**Ein Aussteller stellt einen Nachweis aus -- Ein Benutzer kann ihn bei sich verwahren -- Ein Unabhängiger Verifier kann diesen Überprüfen.**

Dabei spielt es keine Rolle ob es um einen staatlichen oder privatwirtschftlichen Nachweis handelt - das gute alte Papier macht da ja auch keinen Unterschied. Die Prozesse, welche Fach-Sicherheit bieten, liegen ausserhalb dieses Mechanismus, wie in der physischen Realität auch.

**Reply by grt-dti** [reply]
@christianheimann nicht ganz; klar, wir haben in der E-ID keine Probleme wie double spending, dafür aber eine ganze Menge andere, die digitale Eigentumsnachweise nicht haben (räuberische Kopien, langkettige referentielle Integritäten, selective disclosure und vor allem extrem langgenutzte Claims/VC's um nur ein paar zu nennen).
Genau darauf beruht ja mein Argument, die beiden Nachweis-Welten auseinander zu nehmen:
In diesem Ökosystem haben wir einerseits Eigentumsnachweise (etwas das ich besitze und veräussern kann) und Eigenschaftsnachweise (etwas das ich bin und genau nicht veräussern kann).
Nach diesen vielen Diskussionen hier glaube ich, dass diese beiden Welten von Nachweisen differenziert betrachtet werden müssen und für eine dieser Welten haben wir ja längst etablierte Lösungen - folglich sollte man die E-ID Ökosystemidee alleinig auf die "unveräusserbaren Eigenschaftsnachweise des Individuums" ausrichten.

**Reply by grt-dti** [reply]
> Es geht nicht um Vermischung @grt-dti, es geht um die Chance, stets gleiche Abläufe für den Benutzer gleich zu ermöglichen: **Ein Aussteller stellt einen Nachweis aus -- Ein Benutzer kann ihn bei sich verwahren -- Ein Unabhängiger Verifier kann diesen Überprüfen.**
> 
> Dabei spielt es keine Rolle ob es um einen staatlichen oder privatwirtschftlichen Nachweis handelt - das gute alte Papier macht da ja auch keinen Unterschied. Die Prozesse, welche Fach-Sicherheit bieten, liegen ausserhalb dieses Mechanismus, wie in der physischen Realität auch.

In einer Welt, die nur aus Prozessen/Funktionen/Verhalten aber keinen Daten besteht, bin ich bei dir - dummerweise hat diese Welt extrem viele Daten und darum muss die Charakteristik der Daten (Schutzbedürfnis, Ablageorte, Lebenszyklen,...) in die Frage der strukturellen Abgrenzungen mit betrachtet werden. Prozesse sind nicht alles.....

Und DOCH: das Papier macht einen Unterschied - in aller Regel existiert es nur einmal und nur an einem Ort; etwas dass man von elektronisch repräsentierten Informationen (Ketzer würden es "Daten" nennen) genau nicht behaupten kann.

**Comment by KampfCaspar** [comment]
IMHO sollte sich der Bund gar auf die Protokolldefinitionen und ein Referenz-Wallet beschränken - neben der eigentlichen Ausstellung der 'Kern-eID' gemäss ebendiesen Protokolldefinitionen. **Falls** es dann noch eigene Netzwerk-Infrastruktur braucht, gehörte das natürlich dazu - aber am Liebsten wäre mir etwas, was ganz ohne zentrale **physische** Infrastruktur auskommt. z.B. gibt es keinen Grund, wieso die Verbindung von einem Webshop zu meinem Wallet zwingend irgendwas mit Infrastruktur des Bundes zu tun haben müsste.

Das wäre die eID für mich.

**Reply by fforootd** [reply]
> Referenz-Wallet beschränken - neben der eigentlichen Ausstellung der 'Kern-eID' gemäss ebendiesen Protokolldefinitionen.

Der Bund sollte IMO nur die eID herausgeben (und deren Infra anbieten). Ich sehe den Mehrwert alles in ein Ökosystem einzubringen 😁 nicht

**Reply by ii-nik** [reply]
Referenz-Wallet: Guter Punkt. Dazu habe ich noch eine weitere Diskussion hier https://github.com/e-id-admin/governance-sounding-board/discussions/12 angestossen.

**Reply by christianheimann** [reply]
Nach SSI-Grundsätzen soll die Kommunikation von Wallet zu Verifier direkt sein (abgesehen von Netzwerkkomponenten, welche den Nachrichtentransfer ermöglichen, aber nicht mitlesen können aka "Mediator"); der Bund würde die/eine Registry betreiben ([siehe Diskussion #9](https://github.com/e-id-admin/governance-sounding-board/discussions/9)), welche den Vertrauensanker deiner Beweise enthält.

Selbst der abgefragte Registry-Node (Instanz, Rechner ...) erfährt nicht, was genau geprüft wird, sondern nur, wessen "Issuer"-Public-Key geholt wird, und irgend eine, für Aussenstehende zufällige Zeichenkette, zur Überprüfung einer allfälligen Revokation.

**Reply by fforootd** [reply]
Mediatoren sind mit vorsicht zu geniessen. So entfallen an diesen immer noch eine signifikante Menge an Randdaten. Das Problem ist bereits aus Verfahren die auf "double blinding" setzen bekannt und schützt nur zum Teil vor Interessen und Angriffen.

**Comment by pit-77** [comment]
Bei der Festlegung des Nutzens stellt sich primär die Frage, ob die Schweizer E-ID über die Landesgrenzen hinaus Bedeutung haben soll/kann/muss?

Sollen beispielsweise ausländischen Hochschulen Schweizer Professorinnen und Professoren anhand der eidgenössischen E-ID identifizieren oder müssen sich diese eine international anerkannte ID zulegen?

In Deutschland arbeitet ein Konsortium, mit 26 namhaften öffentlichen und privaten Partnern, an einem offenen [Ökosystem](https://idunion.org/), nachfolgend mit wwID bezeichnet (= dynamischer Komplex von Gemeinschaften). Dieses soll der dezentralen Identitätsverwaltung dienen, weltweit nutzbar sein und sich an europäischen Werten und Regularien orientieren. Das System erlaubt natürlichen und juristischen Personen seine Identitätsinformationen selbst zu verwalten und zu entscheiden, wann und mit wem er diese teilen möchte.
https://e-parlament-ch.ch/parlaments-geschaefte/e-id-kann-die-schweiz-mit-der-rasant-voranschreitenden-digitaltechnik-schritt-halten

**Reply by christianheimann** [reply]
Klar gefordert wurde von fast zwei Dritteln der Teilnehmenden der informellen [Konsultation zum "Zielbild E-ID"](https://www.bj.admin.ch/dam/bj/de/data/staat/gesetzgebung/staatliche-e-id/ber-konsultation.pdf), dass eine Kompatibilität respektive Interoperabilität mit europäischen und internationalen digitalen Identitäts-Ökosystemen eine wichtige Eigenschaft darstellt. Die aktuellen Entwicklungen in der EU werden vom Projektteam intensiv verfolgt ([eIDAS 2.0 Proposal](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=COM%3A2021%3A281%3AFIN)); die genauen Voraussetzungen zu einer Interoperabilität sind derzeit aber noch nicht bekannt.
